jobs/JGLOBAL_STAGE_IC

```diff
@@ -3,15 +3,38 @@
 source "${HOMEgfs}/ush/preamble.sh"
 source "${HOMEgfs}/ush/jjob_header.sh" -e "stage_ic" -c "base stage_ic"
 
-# Restart conditions for GFS cycle come from GDAS
+# Define significant cycles
 # shellcheck disable=SC2153
-rCDUMP=${RUN}
-# shellcheck disable=SC2153
-[[ ${RUN} = "gfs" ]] && export rCDUMP="gdas"
-export rCDUMP
+half_window=$(( assim_freq / 2 ))
+current_cycle="${PDY}${cyc}"
+previous_cycle=$(date --utc -d "${PDY} ${cyc} - ${assim_freq} hours" +%Y%m%d%H)
```

Comment:
These are created/defined in `task.py` based on `PDY` and `cyc`.

Reply:
Good to know, will remove from script, thanks!